package.json

```diff
@@ -129,7 +129,7 @@
     "grunt-cli": "latest",
     "grunt-contrib-clean": "latest",
     "grunt-contrib-jshint": "latest",
-    "grunt-mocha-hack": "latest",
+    "grunt-mocha-hack": "git://github.com/mrvisser/grunt-mocha-hack#full-stack-trace",
```

Comment:
Have you found anything about whether we can get rid of `grunt-mocha-hack` and use `grunt-simple-mocha` again?